### PR TITLE
【Hackathon  No.16】add  API RReLU

### PR DESCRIPTION
**Pull request by OccupyMars2030**
### PR types

 New features
### PR changes

APIs
### Describe

【PaddlePaddle Hackathon 2】16. add RReLU activation function to Paddle #40317

### CODE REVIEW

**Comment by dingjiaweiww** [comment]
请先通过CI噢~

**Comment by SmirnovKol** [comment]
/home/smirnov/Documents/paddle_hackathon/tutorials/paddle.seed.png

**Comment by SmirnovKol** [comment]
paddle seed

**Comment by SmirnovKol** [comment]
In the file  test_dropout_op.py ,  once paddle.seed(100) is  set,   the paddle.rand   and  paddle.nn.functional.dropout   will  generate  fixed  random  values,   so  find out what is  the mechanism  behind it   and  use the mechanism  in   rrelu  op.

**Comment by SmirnovKol** [comment]
curandStatePhilox4_32_10_t

**Comment by SmirnovKol** [comment]
study the implementation of  
dropout,
leaky_relu, (and other  activation ops)
paddle.seed,
paddle.uniform
